qiskit/providers/fake_provider/generic_backend_v2.py

```diff
@@ -53,15 +53,15 @@
     "cz": (1e-8, 9e-7, 1e-5, 5e-3),
     "id": (3e-8, 4e-8, 9e-5, 1e-4),
     "rz": (0.0, 0.0),
-    "sx": (1e-8, 9e-7, 1e-5, 5e-3),
-    "x": (1e-8, 9e-7, 1e-5, 5e-3),
+    "sx": (3e-8, 4e-8, 9e-5, 1e-4),
+    "x": (3e-8, 4e-8, 9e-5, 1e-4),
     "measure": (1e-8, 9e-7, 1e-5, 5e-3),
     "delay": (None, None),
     "reset": (None, None),
 }
 
 # Fallback values for gates with unknown noise default ranges.
-_NOISE_DEFAULTS_FALLBACK = (1e-8, 9e-7, 1e-5, 5e-3)
+_NOISE_DEFAULTS_FALLBACK = (3e-8, 4e-8, 9e-5, 1e-4)
```

Comment:
In the spirit of the bugfix, should the fallback have different values for 1q versus multi-q gates?

Reply:
hah, the `_NOISE_DEFAULTS_FALLBACK` were actually not used, so good suggestion in the spirit of the bugfix.